src/Steps/Step.spec.ts

```diff
@@ -0,0 +1,25 @@
+import Step from './Step';
+
+import RolesStore from '../RolesStore';
+
+import AccessControlError from '../Errors/AccessControlError';
+
+import rolesFixture from '../__fixtures__/roles';
+
+describe('Step (base class)', () => {
+  it('throw error if `rolesStore` is not setup when calling `checkRolesExist`', () => {
```

Comment:
Test cases should read like `it('should return false when permission is denied')`.